File: app/src/main/java/ir/food/kitchenAndroid/app/EndPoints.java
package ir.food.kitchenAndroid.app;

import ir.food.kitchenAndroid.BuildConfig;

public class EndPoints {

    /*TODO : check apis and ports before release*/

    public static final String IP =
            "http://happypizza.ir";
//           "http://192.168.1.145";

    public static final String HAKWEYE_IP =

            "http://happypizza.ir";
//           "http://192.168.1.145";


    public static final String APIPort = "3010";

    public static final String HAWKEYE_APIPort = "1890";

    public static final String ACRA_PATH = "http://turbotaxi.ir:6061/api/v1/crashReport";

    public static final String WEBSERVICE_PATH = IP + ":" + APIPort + "/api/kitchen/v1/";

    public static final String HAWKEYE_PATH = HAKWEYE_IP + ":" + HAWKEYE_APIPort + "/api/user/v1/";
    public static final String HAWKEYE_LOGIN_PATH = HAKWEYE_IP + ":" + HAWKEYE_APIPort + "/api/user/v1/login/phone/";

    /******************************** Base Api *********************************/

    public static final String APP_INFO = WEBSERVICE_PATH + "app/info";
    public static final String LOG_IN = WEBSERVICE_PATH + "login";
    public static final String REGISTER_CODE = WEBSERVICE_PATH + "verificationcode";
    public static final String LOGIN_CODE = LOG_IN + "/" + "verificationcode";
    public static final String REGISTER = WEBSERVICE_PATH + "register";
    public static final String ORDER = WEBSERVICE_PATH + "order/";
    public static final String NOT_READY_ORDER = ORDER + "active";
    public static final String READY = ORDER + "ready";
    public static final String HISTORY = ORDER + "finished";

    /******************************** refresh token Api *********************************/

    public static final String REFRESH_TOKEN = HAWKEYE_PATH + "token";
    public static final String VERIFICATION = HAWKEYE_LOGIN_PATH + "verification";
    public static final String CHECK = HAWKEYE_LOGIN_PATH + "check";

}

File: app/src/main/java/ir/food/kitchenAndroid/app/EndPoints.kt
package ir.food.kitchenAndroid.app

class EndPoints {
    companion object {
        /*TODO : check apis and ports before release*/
        const val IP = "http://happypizza.ir:3010"
        const val HAKWEYE_IP = "http://happypizza.ir:1890"
        const val WEBSERVICE_PATH = "$IP/api/kitchen/v1/"
        const val CHECKOUT = "$IP/api/kitchen/v1/checkout/"
        const val DELI_PATH = "$IP/api/kitchen/v1/delivery/"
        const val HAWKEYE_PATH = "$HAKWEYE_IP/api/user/v1/"
        const val HAWKEYE_LOGIN_PATH = "$HAKWEYE_IP/api/user/v1/login/phone/"

        /******************************** Base Api  *****************************************/
        const val APP_INFO = WEBSERVICE_PATH + "app/info"
        const val LOG_IN = WEBSERVICE_PATH + "login"
        const val REGISTER_CODE = WEBSERVICE_PATH + "verificationcode"
        const val LOGIN_CODE = "$LOG_IN/verificationcode"
        const val REGISTER = WEBSERVICE_PATH + "register"
        const val ORDER = WEBSERVICE_PATH + "order/"
        const val NOT_READY_ORDER = ORDER + "active"
        const val READY = ORDER + "ready"
        const val SENDING = ORDER + "ready/sending"
        const val HISTORY = ORDER + "finished"
        const val PACKED = ORDER + "pack"
        const val CANCEL_DELIVER = ORDER + "setFree"
        const val GET_PRODUCTS = WEBSERVICE_PATH + "product"
        const val EDIT_PRODUCTS = WEBSERVICE_PATH + "product"
        const val GET_PRODUCTS_TYPE = WEBSERVICE_PATH + "product/type"
        const val DELI_FINANCIAL = DELI_PATH + "financial"
        const val STATUS_DELI = DELI_PATH + "status"
        const val LOCATION_DELI = DELI_PATH + "location/"
        const val CHECKOUT_DELI = CHECKOUT + "delivery"

        /******************************** refresh token Api  *****************************************/
        const val REFRESH_TOKEN = HAWKEYE_PATH + "token"
        const val VERIFICATION = HAWKEYE_LOGIN_PATH + "verification"
        const val CHECK = HAWKEYE_LOGIN_PATH + "check"
    }
}